### ISSUE TITLE
Remove `ArticleDesign.PrintShop`

### ISSUE THREAD

**Issue by JamieB-gu** [guardian/dotcom-rendering#12256]
This was introduced as a custom design in an attempt to handle the special cases where some print shop pieces[^1] were made immersive without a main media. However, we have decided that this should no longer be handled as a special case. Print shop pieces should work like other articles:

1. If they **have** a main media they can be either showcase or immersive.
2. If they **do not have** a main media they should not be set to immersive.

## Examples

- https://www.theguardian.com/artanddesign/article/2024/jul/31/buy-a-glastonbury-2024-photograph-ice-cream-van-crowd-david-levene

[^1]: https://www.theguardian.com/artanddesign/series/guardian-print-shop

**Comment by paperboyo** [comment]
Just to pt. 2

> If they **do not have** a main media they should not be set to immersive.

“Should not” won’t **ever** result in “won’t”. And, to be honest, it seems to stem **purely** from DCR’s inability to deal with that scenario. Full scenario being not even just the lack of Main media in an Immersive piece, but the fact that the very first thing in a body is an image in an Immersive role ([example](https://www.theguardian.com/artanddesign/2020/dec/08/buy-a-classic-sport-photograph-gazzas-last-minute-joy)) too.

In the spirit of [this comment](https://github.com/guardian/dotcom-rendering/pull/1485#issuecomment-628222332) and Gareth’s

> We should support lack of main media

Main media is not mandatory, never was. Any article, including those with Immersive Display Hint, can lack Main media alright.

FWIW, I never understood why DCR (unlike frontend; although both still struggle with furniture area) had problems with a combination of Immersive sans Main media **and** first body image set to Immersive. (Hats off to users who discovered this combo, as much as it complicates support, not their fault they were given the choice.) I always felt inventing a new `format` for a series is not a good idea. Ideally, platforms could just support whatever is legal. But I do recognise the practicalities… and with CP’s direction/advice, this series may not be the main “offender” any more. But the point of principle stands: both old and new pieces will continue to be able to use whatever is legal and if there is no support, it will look broken. However rarely.

In short: this (Immersive sans Main media [+1st body image immersified]) is not a hack at its core. It’s only a hack (and, as all hacks, flaky and liable to breakage), **because** DCR decided to deal with it in… a hacky way. I am first to admit I don’t understand how complex it might have been to ensure proper support.

So, I am ecstatic to see this `format` die! It was always a weirdo . Making exceptions and providing support for odd series isn’t ever a good idea and it will come and bite us. But the model still allows for flexibility which, ideally, should be dealt with in a generic manner.
